src/Uno.UI/UI/Xaml/Controls/Border/Border.cs

```diff
@@ -96,10 +96,17 @@ public CornerRadius CornerRadius
 			set => this.SetValue(CornerRadiusProperty, value);
 		}
 
-		// Using a DependencyProperty as the backing store for CornerRadius.  This enables animation, styling, binding, etc...
 		public static readonly DependencyProperty CornerRadiusProperty =
-			DependencyProperty.Register("CornerRadius", typeof(CornerRadius), typeof(Border), new PropertyMetadata(CornerRadius.None,
-				(s, e) => ((Border)s)?.OnCornerRadiusChanged((CornerRadius)e.OldValue, (CornerRadius)e.NewValue)));
+			DependencyProperty.Register(
+				nameof(CornerRadius),
+				typeof(CornerRadius),
+				typeof(Border),
+				new FrameworkPropertyMetadata(
+					CornerRadius.None,
+					FrameworkPropertyMetadataOptions.AffectsMeasure | FrameworkPropertyMetadataOptions.AffectsArrange,
```

Comment:
Affecting _measure_ will always affects arrange.
You should also add the option `AutoConvert`